### PR TITLE
Redirect when login

### PR DESCRIPTION
**Pull request by quocthangit247**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class App extends Component {
             <Switch>
                 <Route exact path="/" component={Login} />
                 <Route exact path="/profile" component={Profile} />
-                <Route exact path="/profile-detail" component={ProfileDetails} />
+                {/* <Route exact path="/profile-detail" component={ProfileDetails} /> */}
```
Comment (dongvanhuy) [inline src/App.js]: Why you comment code ProfileDetail Component at here  ?

File: src/containers/login/Login.js [hunk 2]
```diff
@@ -67,10 +77,11 @@ export class Login extends Component {
   }
 
   render() {
-      console.log('>>>>>>>>>>>>>>>>>>>>>>>>', this.state);
+      if (this.state.success === true) {
+          return <Redirect to="/profile" />;
```
Comment (dongvanhuy) [inline src/containers/login/Login.js]: you should use this.props.push('/profile'); with library import { push } from 'react-router-redux';

Comment (quocthangit247) [reply src/containers/login/Login.js]: done !!!

**Comment by dongvanhuy** [comment]
resolve conflict please !

**Pull request merged**